traffic_router/core/src/main/java/org/apache/traffic_control/traffic_router/core/dns/NameServer.java

```diff
@@ -328,11 +328,19 @@ private static RRset setNegativeTTL(final RRset original, final int flags) {
 			if (record instanceof SOARecord) {
 				final SOARecord soa = (SOARecord) record;
 
+				// Set the "minimum" attribute to be the maximum of the current minimum value and 900 (seconds)
+				// This is done to increase the negative caching TTL, so as to maximize the time interval between
+				// successive NXDOMAIN or NXRRSET responses.
+				final long minimum = Math.max(soa.getMinimum(), 900L);
```

Comment:
Rather than hard-coding `900L`, this should be  configuration option, maybe in `server.xml`.

Reply:
Another place it could go is in `applicationContext.xml`